res/values/strings.xml

```diff
@@ -62,6 +62,7 @@
     <string name="menuItem_refreshRoms">Refresh ROMs</string>
     <string name="menuItem_settings">Settings</string>
     <string name="menuItem_globalSettings">Global settings</string>
+    <string name="menuItem_profiles">Profiles</string>
```

Comment:
Only new string needed for the navigation drawer was a category for the different profiles.